app/_data/plugins.yml

```diff
@@ -49,6 +51,14 @@
 - name: Traffic Control
   info: Manage, throttle and restrict inbound and outbound API traffic
   plugins:
+    - name: Edge Caching
+      info: Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+      enterprise: true
+
+    - name: Rate Limiting
+      info: Lorem ipsum dolor sit amet, consectetur adipiscing elit.
```

Comment:
info: Upgrades Kong Community Edition rate limiting with more flexibility and higher performance.